src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/IdeaCollection/Proposal/Proposal.ts

```diff
@@ -302,6 +306,9 @@ export var listItemDirective = (
                     scope.selectedState = "is-not-selected";
                 }
             }));
+            if (typeof scope.processProperties.hasAuthorInListItem === "undefined") {
+                scope.processProperties.hasAuthorInListItem = true;
```

Comment:
I am not convinced that setting defaults here is a good idea. It should be easy to find out what the default values are. I don't know of any better place though